src/verify.cpp

```diff
@@ -1,15 +1,15 @@
-
 #include "scitokens.h"
 
 #include <fstream>
 #include <getopt.h>
 #include <iostream>
+#include <string>
 
 namespace {
 
 const char usage[] =
     "\n"
-    "Syntax: %s [--cred cred_file] TOKEN\n"
+    "Syntax: %s [--cred cred_file] < TOKEN\n"
```

Comment:
Previously TOKEN meant the token itself, but this usage is for a file containing the token.  I think it would be better to say something like TOKENFILE to make it clear that it is reading from a file.

Reply:
ah, yes, you are correct.  I'll fix the usage.

Reply:
Fixed in #161